services/todo-legacy/controllers/todo-controller.ts

```diff
@@ -0,0 +1,43 @@
+import { api, Application, inject } from '@loopback/core';
+import { def } from './todo-controller.api';
+import { TodoRepository } from '../repositories/todo';
+import * as util from 'util';
+
+@api(def)
+export class TodoController {
+  @inject('repositories.todo')
+  repository: TodoRepository;
+
+  constructor() {
+  }
+
+  async getTodo(title?: any) {
+    let filter = title ? { where: { title: title }} : {};
+    return await this.repository.find(null, filter);
+  }
+
+  async getTodoById(id: number) {
+    return await this.repository.find(id);
+  }
+
+  async createTodo(todo) {
+    return await this.repository.create(todo);
+  }
+
+  async updateTodo(id: number, todo: any) {
+    return await this.repository.update(id, todo);
+  }
+
+  async replaceTodo(id: number, todo: any) {
+    return await this.repository.replace(id, todo);
+  }
+
+  async deleteTodo(title?: any) {
+    let filter = title ? { where: { title: title } } : {};
+    return await this.repository.delete(null, filter);
+  }
+
+  async deleteTodoById(id: number) {
+    return await this.repository.delete(id);
+  }
```

Comment:
Do we need Todo in all the names above? Seems redundant since you're in the Todo controller already (ie. getById, deleteById, etc).